### DISCUSSION TITLE
Input source level

### DISCUSSION THREAD

**Discussion by jazzman31** [anthonyalfimov/Multi-Device-Player#10] [Suggestions and ideas]
Hi,
Is it possible to add input source fader with Mute button? Maybe adding an option to add VST plugins.... 
Love the idea and thanks!

Jean

**Comment by anthonyalfimov** [comment]
Hi Jean,
Thank you! Input source fader with a mute button are easy to do, I'll add them when I get a chance.
Hosting VST plugins is a great idea, but it's much more time consuming. I doubt I will be able to add that in the near future.
